modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/sqllogic/ItSqlLogicTest.java

```diff
@@ -171,6 +171,8 @@ public class ItSqlLogicTest extends BaseIgniteAbstractTest {
             + "  rest.port: {}\n"
             + "}";
 
+    private static final int METASTORAGE_IDLE_SYNC_TIME_INTERVAL_MS = 10;
```

Comment:
Please describe in the javadoc why this particular value was chosen.

Reply:
Added an explanation